src/common/tests/unit/all.ts

```diff
@@ -18,6 +18,7 @@ import '../../../dialog/tests/unit/Dialog';
 import '../../../label/tests/unit/Label';
 import '../../../listbox/tests/unit/Listbox';
 import '../../../listbox/tests/unit/ListboxOption';
+import '../../../progress/tests/unit/Progress';
```

Comment:
I think we need a functional test for this component, even if it's just verifying it renders something visible on the page. Otherwise someone will have to retroactively (and painfully) add one.

Reply:
Is there really any point doing a functional for this widget given that its not interactive and is fully covered by unit tests. All we'd be doing is checking that widget-core is working?

Reply:
I didn't think so, as I didn't do one for the addons either (same thing, no interactivity). Definitely open to being convinced, though